src/meltano/cli/select.py

```diff
@@ -85,7 +88,7 @@ async def select(
     """
     try:
         if flags["list"]:
-            await show(project, extractor, show_all=flags["all"])
+            await show(project, extractor, install_plugins, show_all=flags["all"])
```

Comment:
I'm happy to ignore missing coverage for this for now. We have an incoming test that'll cover this in https://github.com/meltano/meltano/pull/8518.

Reply:
Great! I was about to ask if you wanted a test 😅